src/index.html

```diff
@@ -8,6 +8,84 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <div class="stars__star">
+      </div>
```

Comment:
No need to put closing tag on separate line here.
```suggestion
      <div class="stars__star"></div>
```